docs/content/Schema/Reference/cube.mdx

```diff
@@ -631,6 +634,28 @@ cubes:
 It'll generate aliases for members such as `ofacts__count`. `sqlAlias` affects
 all member names including pre-aggregation table names.
 
+### <--{"id" : "Parameters"}--> sqlTable
+
+The `sqlTable` property can be used instead of [`sql`](#parameters-sql) to
+directly specify the name of a table from the data source that will be queried
+by this cube.
+
+<CodeTabs>
+
+```javascript
```

Comment:
Example should follow style guide https://github.com/cubedevinc/cube_style_guide